src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -50,6 +52,11 @@
         "^([A-Za-z0-9+\\/]{4})*([A-Za-z0-9+\\/]{4}|[A-Za-z0-9+\\/]{3}=|[A-Za-z0-9+\\/]{2}==)$"
     );
 
+    /**
+     * Pattern for parsing Signature.
+     */
+    private static final Pattern SIGN = Pattern.compile("[A-Za-z0-9+/]+={0,3}");
```

Comment:
@carlosmiranda Does this actually check that it is a least a multiple of 4 characters?

@llorllale I wonder if we shouldn't go farther than just checking if the string is "kind of" a base 64 string. Zold's ruby implementation sometimes has a data-oriented feel I think and shouldn't we directly try to parse this as a RSA signature instead of just a String? This would mean change the behaviour of `Transaction` to make it smarter. 

What do you both think?

Reply:
@victornoel @llorllale converting this to RSA signature is way out of scope for this task. You might wanna raise a separate ticket instead.

Reply:
@victornoel my own thoughts on this would be we should have a decorator instead. Something like `SignedTransaction` which automatically checks the validity of the signature.

Reply:
@victornoel I don't see anything in `java.security` that we could use for what you're proposing

Reply:
@llorllale yes I see what you mean. Though at one point, we will need to actually parse this for real and validate the signature and so on. So we will have to find a solution (and there are for sure, even using only the java library, albeit maybe limited). I understand it's out of scope of the issue :)